src/PhotoNim.nim

```diff
@@ -57,8 +62,24 @@ proc pfm2png(fileIn, fileOut: string, alpha, gamma: float32) =
     echo fmt"Successfully converted {fileIn} to {fileOut}"
 
 
+proc col_pix(im_tr: ImageTracer, ray: Ray, scenary: World, row, col: int): Color = 
+    # Procedure to decide pixel color (it could be useful to check if scenary len is non zero)
+    let dim = scenary.shapes.len
+    for i in 0..<dim:
+        if fastIntersection(scenary.shapes[i], ray): 
+            let 
+                col1 = (1 - exp(-float32(col + row)))
+                col2 = row/im_tr.image.height
```

Comment:
Here we have some conflicts with master branch: we need to resolve conflict in merge editor once we will merge